gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1441,8 +1447,8 @@ protected void configurePegasusSchemaSnapshotGeneration(Project project, SourceS
           task.dependsOn(generatePegasusSchemaSnapshot);
           task.setCurrentSnapshotDirectory(publishablePegasusSchemaSnapshotDir.toFile());
           task.setPreviousSnapshotDirectory(pegasusSchemaSnapshotDir);
-          task.setCodegenClasspath(project.getConfigurations() .getByName(SCHEMA_ANNOTATION_HANDLER_CONFIGURATION)
-              .plus(project.getConfigurations().getByName(PEGASUS_PLUGIN_CONFIGURATION))
+          task.setCodegenClasspath(project.getConfigurations().getByName(PEGASUS_PLUGIN_CONFIGURATION)
```

Comment:
Does changing the ordering here fix the problem we talked about earlier (related to the wrong pegasus version being picked up in the schema annotation handler configuration)?

Reply:
Yes, it fixes the problem